ui/lib/core/addon/components/list-view.js

```diff
@@ -54,4 +54,13 @@ export default class ListView extends Component {
     const items = pluralize(this.itemNoun);
     return `Your ${items} will be listed here. Add your first ${this.itemNoun} to get started.`;
   }
+
+  // callback from HDS pagination to set the queryParams currentPage
+  get paginationQueryParams() {
+    return (page) => {
+      return {
+        currentPage: page,
```

Comment:
if you keep this to page, then you don't have to change the query param name. currentPage was just specific to KV.